src/convertToObject.js

```diff
@@ -12,7 +12,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const correctStyles = {};
+
+  const styles
+    = sourceString.split(';').map(style =>
+      style.split(':'));
```

Comment:
некорректный перенос, просто обрыв строк
```suggestion
  const styles = sourceString
    .split(';')
    .map(style => style.split(':'));
```